charts/ceph-csi-rbd/templates/provisioner-deployment.yaml

```diff
@@ -64,6 +64,10 @@ spec:
             - "--retry-interval-start=500ms"
             - "--default-fstype={{ .Values.provisioner.defaultFSType }}"
             - "--extra-create-metadata=true"
+            # TODO: enable the feature-gate once error has been handled in nfs.
```

Comment:
you can enable the featuregate in rbd and cephfs, disable only in nfs

Reply:
that's done!